### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HubiHonold**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ ] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] It's submitted to the `develop` branch, _not_ the `master` branch if no hotfix
- [ ] When resolving a specific issue, it's referenced in the PR's description (e.g. `Fix #xxx[,#xxx]`, where "xxx" is the Github issue number)
- [ ] All tests are passing
- [ ] New/updated tests are included
- [ ] Changelog was modified

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,33 @@
-HumHub - Social Network Kit
+HumHub - Putting People and Pieces together
```
Comment (ArchBlood) [inline README.md:1]: I'm not sure about this, maybe something along the lines of;
` HumHub - Bringing People and Communities Together` or `HumHub - Creating Communities Together `?

File: README.md [hunk 2]
```diff
@@ -1,20 +1,33 @@
...
-- Enterprise Social Networks
-- Private Social Networks
+**The software is written in PHP** and is best described by dividing into 4 main parts: **User, Spaces, Content and Modules.**
 
-More information:
-- [Homepage & Demo](http://www.humhub.org)
-- [Documentation & Class Reference](http://docs.humhub.org)
-- [Community](http://community.humhub.com/)
-- [Licence](https://www.humhub.com/licences)
+**User:** All users have their own customisable profile (including name, profile picture, cover photo and personal information) and can follow and interact with each other. If wished and enabled, users can create own content, comment posts and join Spaces. Profile fields, permissions and all settings can be defined individually by the network operator (administrator).
```
Comment (ArchBlood) [inline README.md:12]: Should also mention that the friendship function can also be enabled as well, so around `follow` something should be added here to mention friendships.

**Review by ArchBlood — COMMENTED** [review]
@luke- some things to think about.

File: README.md (hunk 1 above)
Comment (luke-) [reply README.md:1]: We use `Putting People and Pieces together` as claim

File: README.md (hunk 2 above)
Comment (HubiHonold) [reply README.md:12]: There are so many things and functions that could be mentioned. All in all the text should be short and give a good overview of HumHub and it´s functions. But thanks for the feedback, will take a look into it once again soon.

**Pull request merged**